pkgs/by-name/ha/hatch/package.nix

```diff
@@ -74,6 +74,19 @@ python3Packages.buildPythonApplication rec {
     export HOME=$(mktemp -d);
   '';
 
+  pytestFlagsArray = [
+    # AssertionError on the version metadata
+    # https://github.com/pypa/hatch/issues/1877
+    "--deselect=tests/backend/metadata/test_spec.py::TestCoreMetadataV21::test_all"
+    "--deselect=tests/backend/metadata/test_spec.py::TestCoreMetadataV21::test_license_expression"
+    "--deselect=tests/backend/metadata/test_spec.py::TestCoreMetadataV22::test_all"
+    "--deselect=tests/backend/metadata/test_spec.py::TestCoreMetadataV22::test_license_expression"
+    "--deselect=tests/backend/metadata/test_spec.py::TestCoreMetadataV23::test_all"
+    "--deselect=tests/backend/metadata/test_spec.py::TestCoreMetadataV23::test_license_expression"
+    "--deselect=tests/backend/metadata/test_spec.py::TestCoreMetadataV23::test_license_files"
+    "--deselect=tests/backend/metadata/test_spec.py::TestProjectMetadataFromCoreMetadata::test_license_files"
+  ];
```

Comment:
Forgive my ignorance: what's the reason for needing to specify the flags manually instead of using `disabledTests`?
e.g. ``disabledTests = [ "test_all" "test_license_expression" "test_license_files" ]``

Is your approach just more granular?